FileShareAdminClient/FileShareApiAdminClient.cs

```diff
@@ -137,9 +153,20 @@ public async Task CommitBatch(IBatchHandle batchHandle)
             }
         }
 
+        public async Task<IResult<CommitBatchResponse>> CommitBatch(IBatchHandle batchHandle, CancellationToken cancellationToken = default)
```

Comment:
This method isn't covered by any unit tests?

Reply:
Unit test case was written to cover the newly added method CommitBatch, but the parameter CancellationToken got missed. It is fixed now.